File: DheerajKumar/problem25.cpp
//25. WAP to convert a character into its toggle case.

#include<iostream>
using namespace std;

void toggleChar(char character){
    int mapper = static_cast<char>(character);

    if(mapper > 90){
        mapper = mapper - 32;
        cout<<"The Toggle case = "<<static_cast<char>(mapper);
        return;
    }
    if(mapper < 97){
        mapper = mapper + 32;
        cout<<"The Toggle case = "<<static_cast<char>(mapper);
        return;
    }

    cout<<"Error !";
    return;
}

int main(){
    char userInput;
    cout<<"Enter the character ->>";
    cin>>userInput;

    toggleChar(userInput);
    
}

File: LovelyKumari/BasicProgramming/Problem25.java
import java.util.Scanner;

public class Problem25 {
    //WAP to convert a character into its toggle case
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter a character: ");
        char ch=sc.next().charAt(0);

        // conversion of character to its toggle case
        if(Character.isAlphabetic(ch)){
            if(Character.isUpperCase(ch)){
                ch=Character.toLowerCase(ch);
            } else if(Character.isLowerCase(ch)){
                ch=Character.toUpperCase(ch);
            } System.out.println("The toggled case of the entered character is: "+ ch);

        } else {
            System.out.println("The entered character is not a letter.");
        }
        
    }
}
